### PR TITLE
Parameter renames to fix auto generated documentation

### PR DESCRIPTION
**Pull request by atbagga**
devops_organization --> organization
open_browser --> open

### CODE REVIEW

File: azure-devops/azext_devops/dev/pipelines/build.py [hunk 1]
```diff
@@ -20,8 +20,8 @@
 logger = get_logger(__name__)
 
 
-def build_queue(definition_id=None, definition_name=None, branch=None, variables=None, open_browser=False,
```
Comment (gauravsaralMs) [inline azure-devops/azext_devops/dev/pipelines/build.py]: `open_browser` is much more verbose/ explicit.
I would recommend we retain `open_browser` as well and have `open` as one of the alias.
@geverghe  can tell better

Comment (geverghe) [reply azure-devops/azext_devops/dev/pipelines/build.py]: IMO, we should start with just --open;

Comment (gauravsaralMs) [reply azure-devops/azext_devops/dev/pipelines/build.py]: we can go ahead with just `open` but it is not very clear what is `open` in terms of CLI. `open_browser` was looking better to me that it was explicit that we are going to open something in browser.
but as long as we are relying on help text then it is fine

Comment (geverghe) [reply azure-devops/azext_devops/dev/pipelines/build.py]: Noted; we can probably start here and wait for feedback.

**Pull request merged**